ReadMe.rst

```diff
@@ -27,6 +27,95 @@ Conventions
 - Platform CI files should be placed in a `<PlatformPkg>/.azurepipelines` folder in the platform repository.
   - Top level CI files should be named `<HostOs><ToolChainTag>.yml`
 
+GitHub Automation
+=================
+
+This repository also drives automation of Project Mu GitHub repositories.
+
+Leaf Workflows & Reusable Workflows
+-----------------------------------
+
+Workflows are split into two categories **(1) leaf** and **(2) reusable**.
+
+The main reason for reusable workflows is to consolidate the main logic for the workflow to a single file and allow
+the leaf workflow to be present in repositories that opt into what the reusable workflow provides. Leaf workflows can
+provide any repo-specific input to a reusable workflow (if necessary). Leaf workflows can be considered minimal
+wrappers around reusable workflows.
+
+Reusable Workflows
+------------------
+
+These workflow are only designed to be called form other workflows. The files are maintained in the `.github/workflows`
```

Comment:
form should be from

Reply:
Fixed in PR commit https://github.com/microsoft/mu_devops/pull/32/commits/02720e1c4f9c062c376b7c3c5a9b8283f4525b6a